syntax = "proto2";
package wtlogin;

// oicq/wlogin_sdk/pb/device_report.java
message DeviceReport {
  optional string bootloader = 1;
  optional string proc_version = 2;
  optional string codename = 3;
  optional string incremental = 4;
  optional string fingerprint = 5;
  optional string boot_id = 6;
  optional string android_id = 7;
  optional string base_band = 8;
  optional string inner_version = 9;
}

// oicq/wlogin_sdk/pb/sec_trans.java
message SecTransInfo {
  optional string phone_brand = 1;
  optional string model_type = 2;
  optional string wifi_mac = 3;
  optional string bssid = 4;
  optional string os_language = 5;
  optional uint32 qq_language = 6;
  optional string gps_location = 7;
}
